cms/jinja2/templates/base_page.html

```diff
@@ -68,4 +68,12 @@ <h1 class="ons-u-fs-3xl common-header__heading">
 {% block scripts %}
     {{ super() }}
     {% include "templates/partials/content_scripts.html" %}
+
+    <!-- Load JS to refresh user session if required -->
+    {% if AWS_COGNITO_LOGIN_ENABLED and not IS_EXTERNAL_ENV and request.user.is_authenticated %}
+        <script>
+            window.authConfig = {{ AUTH_CONFIG|safe }};
```

Comment:
Issue: This needs to use the `json_script` template tag to avoid escaping issues.

Reply:
Done!